arch/arm/src/stm32l5/hardware/stm32l562xx_rcc.h

```diff
@@ -0,0 +1,867 @@
+#define STM32L5_RCC_AHB2RSTR_OFFSET   0x002c  /* AHB2 peripheral reset register */
+#define STM32L5_RCC_AHB3RSTR_OFFSET   0x0030  /* AHB3 peripheral reset register */
+#define STM32L5_RCC_APB1RSTR1_OFFSET  0x0038  /* APB1 Peripheral reset register 1 */
+#define STM32L5_RCC_APB1RSTR2_OFFSET  0x003c  /* APB1 Peripheral reset register 2 */
+#define STM32L5_RCC_APB2RSTR_OFFSET   0x0040  /* APB2 Peripheral reset register */
+#define STM32L5_RCC_AHB1ENR_OFFSET    0x0048  /* AHB1 Peripheral Clock enable register */
+#define STM32L5_RCC_AHB2ENR_OFFSET    0x004c  /* AHB2 Peripheral Clock enable register */
+#define STM32L5_RCC_AHB3ENR_OFFSET    0x0050  /* AHB3 Peripheral Clock enable register */
+#define STM32L5_RCC_APB1ENR1_OFFSET   0x0058  /* APB1 Peripheral Clock enable register 1 */
+#define STM32L5_RCC_APB1ENR2_OFFSET   0x005c  /* APB1 Peripheral Clock enable register 2 */
+#define STM32L5_RCC_APB2ENR_OFFSET    0x0060  /* APB2 Peripheral Clock enable register */
+#define STM32L5_RCC_AHB1SMENR_OFFSET  0x0068  /* RCC AHB1 low power mode peripheral clock enable register */
+#define STM32L5_RCC_AHB2SMENR_OFFSET  0x006c  /* RCC AHB2 low power mode peripheral clock enable register */
+#define STM32L5_RCC_AHB3SMENR_OFFSET  0x0070  /* RCC AHB3 low power mode peripheral clock enable register */
+#define STM32L5_RCC_APB1SMENR1_OFFSET 0x0078  /* RCC APB1 low power mode peripheral clock enable register 1 */
+#define STM32L5_RCC_APB1SMENR2_OFFSET 0x007c  /* RCC APB1 low power mode peripheral clock enable register 2 */
+#define STM32L5_RCC_APB2SMENR_OFFSET  0x0080  /* RCC APB2 low power mode peripheral clock enable register */
+#define STM32L5_RCC_CCIPR_OFFSET      0x0088  /* Peripherals independent clock configuration register 1 */
+#define STM32L5_RCC_BDCR_OFFSET       0x0090  /* Backup domain control register */
+#define STM32L5_RCC_CSR_OFFSET        0x0094  /* Control/status register */
+#define STM32L5_RCC_CRRCR_OFFSET      0x0098  /* Clock recovery RC register */
+#define STM32L5_RCC_CCIPR2_OFFSET     0x009c  /* Peripherals independent clock configuration register 2 */
+#define STM32L5_RCC_SECCFGR_OFFSET    0x00b8  /* Secure configuration register */
+#define STM32L5_RCC_SECSR_OFFSET      0x00bc  /* Secure status register */
+#define STM32L5_RCC_AHB1SECSR_OFFSET  0x00e8  /* AHB1 security status register */
+#define STM32L5_RCC_AHB2SECSR_OFFSET  0x00ec  /* AHB2 security status register */
+#define STM32L5_RCC_AHB3SECSR_OFFSET  0x00f0  /* AHB3 security status register */
+#define STM32L5_RCC_APB1SECSR1_OFFSET 0x00f8  /* APB1 security status register 1 */
+#define STM32L5_RCC_APB1SECSR2_OFFSET 0x00fc  /* APB1 security status register 2 */
+#define STM32L5_RCC_APB2SECSR_OFFSET  0x0100  /* APB2 security status register */
+
+/* Register Addresses *******************************************************/
+
+#define STM32L5_RCC_CR                (STM32L5_RCC_BASE+STM32L5_RCC_CR_OFFSET)
```

Comment:
formatting  is not consistent

Reply:
Please have a look at https://github.com/apache/incubator-nuttx/commit/66f6a22c40060bd66b0f5fe6053b1dbfb70bf326 (The same commit applies to your next review finding)